### PR TITLE
Update prop-types.js

### PR DESCRIPTION
**Pull request by tjavae**
add 
- keyboardDatetime picker
- format string for date, time, datetime, keyboarddatetime

## Related Issue
Relate the Github issue with this PR using `#`

## Description
Simple words to describe the overall goals of the pull request's commits.

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Impacted Areas in Application
List general components of the application that this PR will affect:

*

## Additional Notes
This is optional, feel free to follow your hearth and write here :)

### CODE REVIEW

File: src/prop-types.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ export const propTypes = {
     searchable: PropTypes.bool,
     sorting: PropTypes.bool,
     title: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
-    type: PropTypes.oneOf(['string', 'boolean', 'numeric', 'date', 'datetime', 'time', 'currency'])
+    type: PropTypes.oneOf(['string', 'boolean', 'numeric', 'date', 'datetime', 'time', 'currency', 'keyboaddatetime']),
```
Comment (emrekara37) [inline src/prop-types.js]: Please update , `keyboaddatetime` to `keyboarddatetime`

Comment (tjavae) [inline src/prop-types.js]: Typo: keyboarddatetime instead of keyboaddatetime

**Comment by mbrn** [comment]
Hi @tjavae ,

Why did you add new data type without implement it.

**Comment by mbrn** [comment]
Please merge #916 #917 and #918

**Pull request closed without merging**